### PR TITLE
Movie shell: form & files

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

**Comment by phpgeekfr** [comment]
@GrandSchtroumpf @fritzschoff Don't forget to tell @xmano if this is causing changes in movie tunnel to check e2e tests

**Comment by GrandSchtroumpf** [comment]
it's not :)

File: apps/financiers/financiers/src/app/marketplace/title/view/view.component.html [hunk 1]
```diff
@@ -15,14 +15,62 @@
...
+        If you are not ready to provide us with uch kind of information, just leave a message.
+      </p>
+    </header>
+    <mat-dialog-content>
+      <form [formGroup]="form" fxLayout="column" (ngSubmit)="sendEmail($event, movie)">
+        <mat-form-field appearance="outline">
+          <mat-label>Subject</mat-label>
+          <mat-select formControlName="subject" required>
+            <mat-option></mat-option>
+            <mat-option value="I’m all set. Ready to invest">I’m all set. Ready to invest</mat-option>
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/marketplace/title/view/view.component.html:46]: seems like you used an accent de gue.

File: libs/movie/src/lib/movie/marketplace/shell/shell.component.html [hunk 2]
```diff
@@ -32,13 +32,44 @@
...
+          </a>
+        </mat-card>
+      </div>
+    </article>
+
+    <article *ngIf="movie.promotional.still_photo.length">
+      <h2>Images</h2>
+      <bf-carousel flex [min]="3">
+        <ng-container *ngFor="let photo of movie.promotional.still_photo">
+          <img carouselItem [col]="4" [ref]="photo" asset="empty_poster.webp" >
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/marketplace/shell/shell.component.html:70]: just a heads up. in the other cols you set it to 3, here you set it to 4.

File: libs/organization/campaign/src/lib/marketplace/financing/financing.component.ts [hunk 3]
```diff
@@ -59,17 +63,21 @@ export class MarketplaceFinancingComponent implements OnInit {
 @Pipe({ name: 'apexBudget' })
 export class ApexBudgetPipe implements PipeTransform {
   transform(budget: Budget) {
+    const data = budgetData.filter(b => !!budget[b.serie]);
+    if (!data.length) return;
```
Comment (fritzschoff) [inline libs/organization/campaign/src/lib/marketplace/financing/financing.component.ts:67]: why not return data.length ? {series: ...} : undefined

File: libs/organization/campaign/src/lib/marketplace/financing/financing.component.ts [hunk 4]
```diff
@@ -59,17 +63,21 @@ export class MarketplaceFinancingComponent implements OnInit {
...
+      colors: data.map(b => b.color),
+      data
     };
   }
 }
 
 @Pipe({ name: 'apexFunding' })
 export class ApexFundingPipe implements PipeTransform {
   transform(fundings: Funding[]) {
+    if (!fundings.length) return;
```
Comment (fritzschoff) [inline libs/organization/campaign/src/lib/marketplace/financing/financing.component.ts:80]: same here donc tu veux

**Review by fritzschoff — APPROVED** [review]

File: libs/movie/src/lib/movie/marketplace/shell/shell.component.html (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/marketplace/shell/shell.component.html:70]: Yep the photo are larger than the files : 
https://app.zeplin.io/project/5de7944cf85691577aacfbb7/screen/5f980ceded1f5e0c6616c9ea

File: libs/organization/campaign/src/lib/marketplace/financing/financing.component.ts (hunk 3 above)
Comment (GrandSchtroumpf) [reply libs/organization/campaign/src/lib/marketplace/financing/financing.component.ts:67]: It would work also.

**Pull request merged**